jenkins/runlocal

```diff
@@ -15,13 +15,27 @@ export PB_SERVER_IMAGE_TAG=${USER}
 # is used by the pod.
 export PB_SERVER_IMAGE_PULL_POLICY=never
 
-# Create an RPM from the current source tree.
+# Create an RPM from the current source tree and double check it exists.
 make -C server/rpm clean rpm
+export RPM_PATH=${HOME}/rpmbuild/RPMS/noarch/pbench-server-*.rpm
+ls -ld ${RPM_PATH}
 
 source /etc/os-release
 
+if [[ ${ID} == "fedora" ]]; then
+    BASE_IMAGE=${ID}:${VERSION_ID}
+elif [[ ${ID} == "centos" ]]; then
+    BASE_IMAGE=stream${VERSION_ID%%.*}:latest
+elif [[ ${ID} == "rhel" ]]; then
+    BASE_IMAGE=ubi${VERSION_ID%%.*}:latest
+else
+    echo "Unsupported local OS, ${ID}:${VERSION_ID}" >&2
+    exit 1
+fi
+export BASE_IMAGE
+
 # Build the canned Pbench Server container from the RPM built above.
-BASE_IMAGE=${ID}:${VERSION_ID} RPM_PATH=${HOME}/rpmbuild/RPMS/noarch/pbench-server-*.rpm server/pbenchinacan/container-build.sh
+server/pbenchinacan/container-build.sh
```

Comment:
I think we need some cleanup: running it a second time, I get errors:
```
Error: adding pod to state: name "pbench-in-a-can_nick" is in use: pod already exists
Error: creating container storage: the container name "pbench-in-a-can_nick-elasticsearch" is already in use by 96ca81a04e67fb2ab87e9e768336d71d2517dbfebdcb052349fdf50c2a3f9761. You have to remove that container to be able to reuse that name: that name is already in use
Error: creating container storage: the container name "pbench-in-a-can_nick-postgres" is already in use by 5d02daa7b7f668e8598823723225cf7aac779dbd459db4e6005e33db5c190723. You have to remove that container to be able to reuse that name: that name is already in use
Error: creating container storage: the container name "pbench-in-a-can_nick-keycloak" is already in use by ea7fbafe75a8595a72fa34ee7fd3b8a79a43d047a65637d6052749d1a1f0819e. You have to remove that container to be able to reuse that name: that name is already in use

Keycloak connection successful on : http://localhost:8090

Error: creating container storage: the container name "pbench-in-a-can_nick-pbenchserver" is already in use by a32de0d1ffc2a134618190af4a2d4ee0504c72af3e0c376bd1c2789df81d5bd9. You have to remove that container to be able to reuse that name: that name is already in use
Failure to start Pbench-in-a-Can pod
```

Reply:
Yes, you have to stop the pod running and remove the pbench server container after you run `jenkins/runlocal`.

Reply:
Latest commits now support passing `--cleanup` along to the `jenkins/run-server-func-tests` script.